dashboard/app/entities.go

```diff
@@ -277,6 +277,8 @@ type Commit struct {
 	AuthorName string
 	CC         string `datastore:",noindex"` // (|-delimited list)
 	Date       time.Time
+	Repo       string
```

Comment:
I don't see any code that stores these fields to DB.
What do we do for old records that don't have this?

Reply:
We already should have this info. If you look at fixed bugs, fix commits have links, which means we know repo:
https://syzkaller.appspot.com/bug?extid=8982e75c2878b9ffeac5

Reply:
Hm.. "git://syzkaller.org" confused me and I decided that link was to the syzkaller repo, not kernel.
Let's go https://github.com/google/syzkaller/pull/3994 to simplify communication history.